src/app/lib/utilities/passport/index.js

```diff
@@ -0,0 +1,31 @@
+import pathOr from 'ramda/src/pathOr';
+
+export const getPassportHome = pageData => {
+  const homeServiceUrl = pathOr(
+    null,
+    ['metadata', 'passport', 'home'],
+    pageData,
+  );
+  return pageData && homeServiceUrl
+    ? homeServiceUrl
+        .split('/')
+        .slice(-1)
+        .pop()
+        .toLowerCase()
+    : null;
+};
+
+export const isValidPassportHome = (
```

Comment:
This function is a little hard to read and should make more use of default argument values rather than e.g. `passportHome || ''`.

Something along the lines of this would reduce the cognitive load needed for developers to parse and understand what this function is doing.

```js
export const isValidPassportHome = (
  passportHome = '',
  service = '',
  passportHomesOverride = []
) => {
  const isMissingRequiredArgs = !passportHome && !passportHomesOverride.length;
  if (isMissingRequiredArgs) return true;

  const matchesPassportHome = (string = '') =>
    string.toLowerCase() === passportHome.toLowerCase();

  if (matchesPassportHome(service)) return true;

  return passportHomesOverride.some(matchesPassportHome);
};

```